offchain-modules/packages/x/src/handlers/ckb.ts

```diff
@@ -353,16 +363,18 @@ export class CkbHandler {
     }
     if (confirmed && this.role === 'collector') {
       const unlockRecord = unlockRecords[0];
-      try {
-        const asset = getAsset(unlockRecord.chain, unlockRecord.asset);
-        const fee = asset.getBridgeFee('out');
-        if (BigInt(unlockRecord.amount) <= BigInt(fee)) {
-          throw new Error(`unlock record amount ${unlockRecord.amount} low than fee ${fee}`);
+      if (unlockRecord.chain != ChainType.CARDANO) {
```

Comment:
We should implement the `getBridgeFee` interface for Ada asset instead of using the special case here.